joplin/static/js/admin.js

```diff
@@ -12,20 +12,24 @@ const anchors = {
   "id_process_steps-description": "#step-description",
   "id_process_steps-overview_steps": "#step-steps",
   "id_process_steps-detailed_content": "#step-details",
-  "id_process_steps-quote": "#step-quote",
+  "id_process_steps-quote": "#step-quote"
```

Comment:
Same thing I said in the other PR, but I think we should be wrapping our JS in either a document.ready or IIFE so that we don't get burned by global scopes or eager browsers.